### DISCUSSION TITLE
64-bit Freeplane

### DISCUSSION THREAD

**Discussion by Kitty131** [freeplane/freeplane#239] [Help]
All - 

Is there a 64-bit version of Freeplane?  Thank you for your help!

**Comment by inglor** [comment]
How do you mean? Freeplane is java application.

**Reply by Kitty131** [reply]
Hello - Thank you so much for writing back.  I think my reply will show just how little I know.  

Windows says that the Freeplane I downloaded, v 1.9.10, is a 32-bit program.  This is a small issue for me for reasons that will probably never bother anybody else.  So it's okay if there is only one (32-bit version).  I am grateful that such an amazing program is free and that people continually improve it.

**Reply by dpolivaev** [reply]
Freeplane should run under 64 bit Java. Because freeplane.exe is just a launcher, it does not matter if it is 32 bit or 64 bit itself.
Could you please attach a screen shot showing which file is reported to be a 32 bit program?

**Comment by Kitty131** [comment]
Hello, Dimitry -

Sorry it has taken me a while to write back.  Freeplane.exe is indicated as
32-bit.  Given your explanation I think I better understand what Leonidas
said earlier, "How do you mean? Freeplane is java application."

So Freeplane.exe launches the Java code that is the bulk of Freeplane.
 And since I have 64-bit Java, Freeplane is essentially a 64-bit program
for me.

Even though they don't show much, I have attached two screen shots, one
from Task Manager and one from Process Explorer.  Thank you all so much for
your help!

With Best Regards,

Waqar


[image: Task Manager_Freeplane.jpg][image: Process Explorer_Freeplane.jpg]
